src/zui/ZUIAvatar/index.tsx

```diff
@@ -5,6 +5,7 @@ import { forwardRef } from 'react';
 interface ZUIAvatarProps {
   orgId: number;
   personId: number;
+  isUserAvatar?: boolean;
```

Comment:
As mentioned elsewhere, I think this is problematic because it uses a boolean to not just _tweak_ functionality, but actually change the _role_ of the component. For a user avatar, the `orgId` and `personId` are irrelevant, but those properties are still here (and still used) which is a hint to me that this is not a good solution.